filament/backend/test/BackendTest.cpp

```diff
@@ -178,5 +178,65 @@ int runTests() {
     return RUN_ALL_TESTS();
 }
 
+void getPixelInfo(PixelDataFormat format, PixelDataType type, size_t& outComponents, int& outBpp) {
+    switch (format) {
+        case PixelDataFormat::UNUSED:
+        case PixelDataFormat::R:
+        case PixelDataFormat::R_INTEGER:
+        case PixelDataFormat::DEPTH_COMPONENT:
+        case PixelDataFormat::ALPHA:
+            outComponents = 1;
+            break;
+        case PixelDataFormat::RG:
+        case PixelDataFormat::RG_INTEGER:
+        case PixelDataFormat::DEPTH_STENCIL:
+            outComponents = 2;
+            break;
+        case PixelDataFormat::RGB:
+        case PixelDataFormat::RGB_INTEGER:
+            outComponents = 3;
+            break;
+        case PixelDataFormat::RGBA:
+        case PixelDataFormat::RGBA_INTEGER:
+            outComponents = 4;
+            break;
+    }
+
+    outBpp = outComponents;
+    switch (type) {
+        case PixelDataType::COMPRESSED:
```

Comment:
Isn't the compressed case more complicated and depends on the compression format?

Reply:
Yeah, I adapted this from `PixelBufferDescriptor::computeDataSize` so the `COMPRESSION` case is just to silence warnings.

Reply:
I'll add an assert for this.